docs/sources/next/javascript-api/k6-experimental/browser/elementhandle/isvisible.md

```diff
@@ -0,0 +1,48 @@
+---
+title: 'isVisible()'
+excerpt: 'Browser module: elementHandle.isVisible() method'
+---
+
+# isVisible()
+
+Checks if the element is `visible`.
+
+### Returns
+
+| Type | Description                                       |
+| ---- | ------------------------------------------------- |
+| bool | `true` if the element is `visible`, else `false`. |
```

Comment:
Small suggestion, feel free to discard it if you think the current form is OK:
```suggestion
| bool | `true` if the element is `visible`, `false` otherwise. |
```